lib/Installation/Partitioner/LibstorageNG/v4_3/ConfirmationWarning.pm

```diff
@@ -34,4 +34,22 @@ sub press_yes {
     return $self->{btn_yes}->click();
 }
 
+=head2 is_shown
+
+  is_shown();
+
+Method to check if the warning popup is shown. Warning classes should implement it to use.
+
+For example:
+
+sub is_shown {
+    my ($self) = @_;
+    $self->{lbl_warning}->exist();
```

Comment:
I didn't notice this in previous PRs, but should be 'exists()' for what I could check in other frameworks. We can change it later of course.

Reply:
I also noticed that thing, when used. I agree that it should be `exists`. I'll introduce the change in another PR to not mix-up things and to not make the PR overloaded with the things that do not directly related to the PR.